compose.yaml

```diff
@@ -35,6 +35,7 @@ services:
       UPSTREAM_REAL_IP_ADDRESS: ${UPSTREAM_REAL_IP_ADDRESS:-127.0.0.1}
       UPSTREAM_REAL_IP_HEADER: ${UPSTREAM_REAL_IP_HEADER:-X-Forwarded-For}
       UPSTREAM_REAL_IP_RECURSIVE: ${UPSTREAM_REAL_IP_RECURSIVE:-off}
+      PROXY_READ_TIMOUT: ${PROXY_READ_TIMOUT:-120}
```

Comment:
Does increasing this alone really help when gunicorn timeout is 120 seconds?

https://github.com/frappe/frappe_docker/blob/45c6401da9083b4b53dc7660c5303f339e77e73e/images/worker/Dockerfile#L123

Reply:
https://discuss.erpnext.com/t/erpnext-server-keeps-timing-out/90555/7

gunicorn can be overriden at runtime